malariagen_data/anoph/karyotype.py

```diff
@@ -0,0 +1,112 @@
+import pandas as pd  # type: ignore
+from pandas import CategoricalDtype
+import numpy as np  # type: ignore
+import allel  # type: ignore
+
+from numpydoc_decorator import doc
+from ..util import check_types, _karyotype_tags_n_alt
```

Comment:
If `_karyotype_tags_n_alt` is only used in this module, perhaps it should be moved here.